tutorials/phasorpy_components.py

```diff
@@ -0,0 +1,101 @@
+"""
+Component analysis
+===========
+
+An introduction to component analysis in the phasor space. The
+:py:func:`phasorpy.phasor.phasor_from_lifetime` function is used to calculate
+phasor coordinates as a function of frequency, single or multiple lifetime
+components, and the pre-exponential amplitudes or fractional intensities of the
+components.
+
+"""
+
+# %%
+# Import required modules, functions, and classes:
+
+import matplotlib.pyplot as plt
+import numpy
+
+from phasorpy.components import two_fractions_from_phasor
+from phasorpy.phasor import phasor_from_lifetime
+from phasorpy.plot import PhasorPlot
+
+# %%
+# Fractions of combination of two components
+# ------------------
+#
+# A phasor that lies in the line between two components with 0.25 contribution
+# of the first components and 0.75 contribution of the second component:
+
+frequency = 80.0
+components_lifetimes = [8.0, 1.0]
+real, imag = phasor_from_lifetime(
+    frequency, components_lifetimes, [0.25, 0.75]
+)
+components_real, components_imag = phasor_from_lifetime(
+    frequency, components_lifetimes
+)
+plot = PhasorPlot(
+    frequency=frequency, title='Phasor lying on the line between components'
+)
+plot.plot(components_real, components_imag, fmt='o-')
```

Comment:
@cgohlke I am not sure what you mean with this, can you give me an example?

Reply:
Never mind. I meant to add the fractions as text next to the line segments. It would be nice to have that feature added to the `PhasorPlot.components` methods (not in this PR).